docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -10,371 +9,175 @@ to save power and free CPU resources.
+### Automatic QoS Feature on Windows*
+
+Starting with 2021.4.1 release of OpenVINO and 03.00.00.1363 version of Windows* GNA driver, a new execution mode `ov::intel_gna::ExecutionMode::HW_WITH_SW_FBACK` is introduced
+to assure that workloads satisfy real-time execution. In this mode, the GNA driver automatically falls back on CPU for a particular infer request
+if the HW queue is not empty, so there is no need for explicitly switching between GNA and CPU.
 
 @sphinxdirective
 .. tab:: C++
 
-   ``KEY_GNA_EXEC_TARGET``,  ``KEY_GNA_COMPILE_TARGET``
-
-.. tab:: Python
+   .. doxygensnippet:: docs/snippets/gna/configure.cpp
+      :language: cpp
+      :fragment: [include]
 
-   ``GNA_EXEC_TARGET``,  ``GNA_COMPILE_TARGET``
+   .. doxygensnippet:: docs/snippets/gna/configure.cpp
+      :language: cpp
+      :fragment: [ov_gna_exec_mode_hw_with_sw_fback]
 
-@endsphinxdirective
+.. tab:: Python
 
-## Drivers and Dependencies
+   .. doxygensnippet:: docs/snippets/gna/configure.py
+      :language: python
+      :fragment: [import]
 
-Intel® GNA hardware requires a driver to be installed on the system.
+   .. doxygensnippet:: docs/snippets/gna/configure.py
+      :language: python
+      :fragment: [ov_gna_exec_mode_hw_with_sw_fback]
 
-* Linux\* OS:
-[Download Intel® GNA driver for Ubuntu Linux 18.04.3 LTS (with HWE Kernel version 5.4+)](https://storage.openvinotoolkit.org/drivers/gna/)
+@endsphinxdirective
 
-* Windows\* OS:
-Intel® GNA driver for Windows is available through Windows Update\*
+> **NOTE**: Due to the "first come - first served" nature of GNA driver and the QoS feature, this mode may lead to increased CPU consumption
+if there are several clients using GNA simultaneously.
+Even a lightweight competing infer request which has not been cleared at the time when the user's GNA client process makes its request,
+can cause the user's request to be executed on CPU, thereby unnecessarily increasing CPU utilization and power.
 
-## <a name="models-and-layers-limitations">Models and Layers Limitations</a>
+## Supported inference data types
 
-Because of specifics of hardware architecture, Intel® GNA supports a limited set of layers, their kinds and combinations.
-For example, you should not expect the GNA Plugin to be able to run computer vision models, except those specifically adapted for the GNA Plugin, because the plugin does not fully support 2D convolutions.
+Intel® GNA essentially operates in the low-precision mode which represents a mix of 8-bit (`i8`), 16-bit (`i16`), and 32-bit (`i32`) integer computations.
 
-For the list of supported layers, see the **GNA** column of the **Supported Layers** section in [Supported Devices](Supported_Devices.md).
+GNA plugin users are encouraged to use the [Post-Training Optimization Tool](@ref pot_README) to get a model with quantization hints based on statistics for the provided dataset.
```

Comment:
Refer to the second person in the technical documentation: Use the [Post-Training Optimization Tool](@ref pot_README) to get a model with quantization hints based on statistics for the provided dataset.